xiaogpt/config.py

```diff
@@ -97,7 +97,7 @@ def from_options(cls, options: argparse.Namespace) -> Config:
         for key, value in vars(options).items():
             if value is not None and key in config.__dataclass_fields__:
                 setattr(config, key, value)
-        if not config.openai_key:
+        if (not config.openai_key) and (config.bot != "edgegpt"):
```

Comment:
```suggestion
        if not config.openai_key and config.bot != "edgegpt":
```

The expression is clear enough without parentheses.